lists/ir.csv

```diff
@@ -969,3 +969,4 @@ http://www.asianangels.com,PORN,Pornography,2017-09-05,OONI,Found to be blocked
 http://www.linay.com,PROV,Provocative Attire,2017-09-05,OONI,Found to be blocked in Iran based on OONI tests
 http://www.bbcpersian.com,NEWS,News Media,2017-07-19,BBC,
 https://www.radiofarda.com/,NEWS,News Media,2017-09-29,OONI,
+http://hdl.handle.net/10568/91553,SRCH,Search Engines,2018-04-01,,Handle.net provides persistent identifiers for items in academic repositories
```

Comment:
Can you add the author column? You can put `anon` if you don't want to be identified.